pkg/kubelet/kubelet_node_status.go

```diff
@@ -478,21 +478,39 @@ func (kl *Kubelet) tryUpdateNodeStatus(tryNumber int) error {
 	if tryNumber == 0 {
 		util.FromApiserverCache(&opts)
 	}
-	node, err := kl.heartbeatClient.CoreV1().Nodes().Get(context.TODO(), string(kl.nodeName), opts)
+	originalNode, err := kl.heartbeatClient.CoreV1().Nodes().Get(context.TODO(), string(kl.nodeName), opts)
 	if err != nil {
 		return fmt.Errorf("error getting node %q: %v", kl.nodeName, err)
 	}
-
-	originalNode := node.DeepCopy()
 	if originalNode == nil {
 		return fmt.Errorf("nil %q node object", kl.nodeName)
 	}
 
+	node, changed := kl.updateNode(originalNode)
+
+	if kl.clock.Since(kl.lastStatusReportTime) < kl.nodeStatusReportFrequency && !changed {
+		kl.markVolumesFromNode(node)
+		return nil
+	}
+
+	updatedNode, err := kl.patchNode(originalNode, node)
+	if err == nil {
+		kl.markVolumesFromNode(updatedNode)
```

Comment:
```suggestion
	// If update finishes successfully, mark the volumeInUse as reportedInUse to indicate
 	// those volumes are already updated in the node's status
		kl.markVolumesFromNode(updatedNode)
```

I suggest to keep the comments

Reply:
This comment was moved to the top of markVolumesFromNode.

Reply:
oh, thanks, sorry